easybuild/easyblocks/t/trinity.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, *args, **kwargs):
         EasyBlock.__init__(self, *args, **kwargs)
 
         self.build_in_installdir = True
+        self.path = ''
```

Comment:
initialise to `None` (so it's not prematurely used by accident without causing errors), rename to `self.trinity_subdir`